### DISCUSSION TITLE
Icon/button to jump to top/bottom

### DISCUSSION THREAD

**Discussion by mello7tre** [spacecowboy/Feeder#287] [Ideas]
Hi, as first: great app, as second:
would be nice than when you begin to scroll up/down would appear an icon/button to jump directly to top/bottom.

Best regards, Alberto.

**Comment by Amarok24** [comment]
This is a must-have feature. I'm really surprised why it wasn't implemented from the beginning.

Intuitively (in many apps) a touch/click on the top bar (in this case it's the feed name/tag) lets you jump to the very first article. In the case of Feeder there is even no scroll bar on the right side, making it impossible to know where I am located in the feed, so it's quite annoying to always have to scroll all the way up manually.
